Extensions/AnchorBehavior/anchorruntimebehavior.ts

```diff
@@ -81,265 +96,306 @@ namespace gdjs {
+      } else if (this._rightEdgeAnchor === HorizontalAnchor.WindowCenter) {
+        this._rightEdgeDistance = bottomRightPixel[0] - parentCenterX;
+      }
 
-        // Bottom edge
-        if (this._bottomEdgeAnchor === VerticalAnchor.WindowTop) {
-          this._bottomEdgeDistance = bottomRightPixel[1] - parentMinY;
-        } else if (this._bottomEdgeAnchor === VerticalAnchor.WindowBottom) {
-          this._bottomEdgeDistance = bottomRightPixel[1] - parentMaxY;
-        } else if (this._bottomEdgeAnchor === VerticalAnchor.Proportional) {
-          this._bottomEdgeDistance =
-            (bottomRightPixel[1] - parentMinY) / parentHeight;
-        } else if (this._bottomEdgeAnchor === VerticalAnchor.WindowCenter) {
-          this._bottomEdgeDistance = bottomRightPixel[1] - parentCenterY;
-        }
+      // Bottom edge
+      if (this._bottomEdgeAnchor === VerticalAnchor.WindowTop) {
+        this._bottomEdgeDistance = bottomRightPixel[1] - parentMinY;
+      } else if (this._bottomEdgeAnchor === VerticalAnchor.WindowBottom) {
+        this._bottomEdgeDistance = bottomRightPixel[1] - parentMaxY;
+      } else if (this._bottomEdgeAnchor === VerticalAnchor.Proportional) {
+        this._bottomEdgeDistance =
+          (bottomRightPixel[1] - parentMinY) / parentHeight;
+      } else if (this._bottomEdgeAnchor === VerticalAnchor.WindowCenter) {
+        this._bottomEdgeDistance = bottomRightPixel[1] - parentCenterY;
+      }
 
-        this._invalidDistances = false;
-      } else {
-        //Move and resize the object if needed
-        let leftPixel = 0;
-        let topPixel = 0;
-        let rightPixel = 0;
-        let bottomPixel = 0;
-
-        // Left edge
-        if (this._leftEdgeAnchor === HorizontalAnchor.WindowLeft) {
-          leftPixel = parentMinX + this._leftEdgeDistance;
-        } else if (this._leftEdgeAnchor === HorizontalAnchor.WindowRight) {
-          leftPixel = parentMaxX + this._leftEdgeDistance;
-        } else if (this._leftEdgeAnchor === HorizontalAnchor.Proportional) {
-          leftPixel = parentMinX + this._leftEdgeDistance * parentWidth;
-        } else if (this._leftEdgeAnchor === HorizontalAnchor.WindowCenter) {
-          leftPixel = parentCenterX + this._leftEdgeDistance;
-        }
+      this._hasJustBeenCreated = false;
+    }
 
-        // Top edge
-        if (this._topEdgeAnchor === VerticalAnchor.WindowTop) {
-          topPixel = parentMinY + this._topEdgeDistance;
-        } else if (this._topEdgeAnchor === VerticalAnchor.WindowBottom) {
-          topPixel = parentMaxY + this._topEdgeDistance;
-        } else if (this._topEdgeAnchor === VerticalAnchor.Proportional) {
-          topPixel = parentMinY + this._topEdgeDistance * parentHeight;
-        } else if (this._topEdgeAnchor === VerticalAnchor.WindowCenter) {
-          topPixel = parentCenterY + this._topEdgeDistance;
-        }
+    private _followAnchor(instanceContainer: gdjs.RuntimeInstanceContainer) {
```

Comment:
We could probably split it into `_followVerticalAnchor` and `_followHorizontalAnchor` to avoid useless calculus, especially because game window dimension usually only changes on one axis.